src/components/Global/PageTitle.vue

```diff
@@ -8,7 +8,12 @@
 <script>
 export default {
   name: 'PageTitle',
-  props: ['description'],
+  props: {
```

Comment:
How does everyone feel about this? Adding type to props?

Reply:
I personally like it. It's nice that the errors can be provided to the user if they don't pass an required prop or the right type.